sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GalleryApplicationVersionPublishingProfile.cs

```diff
@@ -59,13 +59,17 @@ public GalleryApplicationVersionPublishingProfile()
         /// <param name="targetExtendedLocations">The target extended locations
         /// where the Image Version is going to be replicated to. This property
         /// is updatable.</param>
+        /// <param name="advancedSettings">Optional. Additional settings to
+        /// pass to the VMApp extension. For advanced used only.</param>
         /// <param name="enableHealthCheck">Optional. Whether or not this
         /// application reports health.</param>
-        public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, IList<TargetRegion> targetRegions = default(IList<TargetRegion>), int? replicaCount = default(int?), bool? excludeFromLatest = default(bool?), System.DateTime? publishedDate = default(System.DateTime?), System.DateTime? endOfLifeDate = default(System.DateTime?), string storageAccountType = default(string), string replicationMode = default(string), IList<GalleryTargetExtendedLocation> targetExtendedLocations = default(IList<GalleryTargetExtendedLocation>), UserArtifactManage manageActions = default(UserArtifactManage), bool? enableHealthCheck = default(bool?))
+        public GalleryApplicationVersionPublishingProfile(UserArtifactSource source, IList<TargetRegion> targetRegions = default(IList<TargetRegion>), int? replicaCount = default(int?), bool? excludeFromLatest = default(bool?), System.DateTime? publishedDate = default(System.DateTime?), System.DateTime? endOfLifeDate = default(System.DateTime?), string storageAccountType = default(string), string replicationMode = default(string), IList<GalleryTargetExtendedLocation> targetExtendedLocations = default(IList<GalleryTargetExtendedLocation>), UserArtifactManage manageActions = default(UserArtifactManage), UserArtifactSettings settings = default(UserArtifactSettings), IDictionary<string, string> advancedSettings = default(IDictionary<string, string>), bool? enableHealthCheck = default(bool?))
```

Comment:
this will require some customization methods in the Customization folder.

Reply:
See our documentation about creating customizations: https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor

Reply:
Added a custom constructor